### ISSUE TITLE
[BUG] <title>多图单轮对话全量SFT，loss 变为零

### ISSUE THREAD

**Issue by Evan9978** [OpenBMB/MiniCPM-o#679]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

训练过程中 loss 突然降为零
<img width="461" alt="截屏2024-11-25 12 16 11" src="https://github.com/user-attachments/assets/32e0b7ce-7178-44ea-b8d6-96295c4c4929">

### 运行环境 | Environment

```Markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):
```

**Comment by wuzhanglin123** [comment]
我也遇到了单轮对话全量SFT，loss 变为零的情况，8卡训练会一直出现，4卡训练基本不出现

**Comment by MickeyFei** [comment]
遇到了单论对话lora sft，loss 从step2 开始就为0的情况 8卡训练出现，单卡训练不出现

**Comment by Evan9978** [comment]
把 tune vision 设置为 false 之后这个问题不再出现了

**Comment by LDLINGLINGLING** [comment]
有没有可能是设置的有效长度太短，导致loss都不计算了

**Comment by Evan9978** [comment]
> 有没有可能是设置的有效长度太短，导致loss都不计算了

这种情况应该不会后面loss全都变零

**Comment by HenryHZY** [comment]
Changing full fine-tuning to lora fine-tuning works for me.
